src/main/App.js

```diff
@@ -159,7 +157,10 @@ class App extends React.Component {
     // The app should only keep tabs on the connection status of the connection manager for the current network
     this.connectionManagers["stagenet"].startCheckingConnection();  // automatically run checkConnection() at regular intervals
     // Check the connection NOW to make sure this.state.isConnectedToDaemon is INITIALLY true if the connection is up
-    this.initialState.isConnectedToDaemon = this.connectionManagers["stagenet"].isConnected(); 
+    let isConnected = this.connectionManagers["stagenet"].isConnected();
```

Comment:
This will always use stagenet and not the selected network type.

Reply:
Incorrect. When a user changes the network, the app stops checking connection on the previously selected network and starts checking on the newly selected network.

https://github.com/BeanTosser/monerostresstester.com/blob/9d12858c23ee8c54a33652cee178a25dc0a46691/src/main/App.js#L791